Core/source/test.ts

```diff
@@ -45,13 +45,15 @@ ava.default('generateKey', async t => {
       variant: 'curve25519', passphrase: 'riruekfhydekdmdbsyd',
       userIds: [{ email: 'a@b.com', name: 'Him' }]
     }));
+  // tslint:disable:no-unsafe-any
```

Comment:
`no-unsafe-any` should not apply to tests. Can you exclude all tests from this rule?

Reply:
Alternatively, can put no-unsafe-any exception on top of every test file

Reply:
Not sure how to disable specific rule for the specific file. Yes, I think it's easier to disable rule right in the file.